.pre-commit-config.yaml

```diff
@@ -1,6 +1,3 @@
-default_language_version:
-    python: python3.7
```

Comment:
Why remove it completely? What I suggested was to put python3 instead of python3.7, so in general any python3 version installed by the user will work, because otherwise if the user does not have python3.7 specifically then the pre-commit will not work.

Reply:
Sorry, I removed that while I was debugging the errors on pre-commit but forgot to add on commit next. Now I've added & tested again. Also adding more hooks to check.